sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/ApplicationDetailsForCOE.vue

```diff
@@ -135,6 +136,12 @@ export default defineComponent({
       initialData.value.disbursementDate = dateOnlyLongString(
         initialData.value.disbursementDate,
       );
+      initialData.value.disabilityProfileStatusDescription =
+        disabilityStatusToDisplay(initialData.value.disabilityProfileStatus);
+      initialData.value.disabilityApplicationStatusDescription =
+        ApplicationDisabilityStatusDescription[
```

Comment:
Is it not the case for `ApplicationDisabilityStatusDescription` to follow the same composable pattern from `disabilityStatusToDisplay`?

Reply:
ok, changed to composable similar to `disabilityStatusToDisplay`.